### PR TITLE
Fix/178

### LINKED ISSUE #178: Borken from block method
force-fields are not subscript able like here:

https://github.com/marrink-lab/polyply_1.0/blob/e26c0c6a286c75e247d8153238360e087226452c/polyply/src/meta_molecule.py#L285-L295

### PR DESCRIPTION
**Pull request by fgrunewald**
Small bug fix & some updates to doc-strings. This fixes #178

### CODE REVIEW

File: polyply/src/meta_molecule.py [hunk 1]
```diff
@@ -217,8 +217,21 @@ def _block_graph_to_res_graph(block):
...
+        force_field: :class:`vermouth.forcefield.ForceField`
+            the force-field that must contain the block
+        monomers: list[:class:`polyply.meta_molecule.Monomer`]
+            a list of Monomer tuples
+        mol_name: str
+            name of the molecule
+
+        Returns
+        -------
+        :class:`polyply.src.meta_molecule.MetaMolecule`
```
Comment (jbarnoud) [inline polyply/src/meta_molecule.py:234]: Is this how you access `MetaMolecule`? Would the following work?

```python
import polyply
polyply.src.meta_molecule.MetaMolecule
```

It looks like the `src` in the path should not be there.

Comment (jbarnoud) [inline polyply/src/meta_molecule.py:234]: Is this how you access `MetaMolecule`? Would the following work?

```python
import polyply
polyply.src.meta_molecule.MetaMolecule
```

It looks like the `src` in the path should not be there.

Comment (jbarnoud) [reply polyply/src/meta_molecule.py:234]:
```suggestion
        :class:`polyply.MetaMolecule`
```

After a quick check, the `src` is correct, though it does not seem to be the way a user would import the object. See my code suggestion.

File: polyply/src/meta_molecule.py [hunk 2]
```diff
@@ -285,10 +311,21 @@ def from_itp(cls, force_field, itp_file, mol_name):
...
+        Parameters
+        ----------
+        force_field: :class:`vermouth.forcefield.ForceField`
+            the force-field that must contain the block
+        mol_name: str
+            name of the block matching a key in ForceField.blocks
+
+        Returns
+        -------
+        :class:`polyply.src.meta_molecule.MetaMolecule`
```
Comment (jbarnoud) [inline polyply/src/meta_molecule.py:325]:
```suggestion
        :class:`polyply.MetaMolecule`
```

Comment (jbarnoud) [inline polyply/src/meta_molecule.py:325]:
```suggestion
        :class:`polyply.MetaMolecule`
```

**Review by jbarnoud — APPROVED** [review]
Just a very minor doc thing.

**Review by jbarnoud — APPROVED** [review]
Github is eating my messages.

File: polyply/src/meta_molecule.py (hunk 2 above)
Comment (fgrunewald) [reply polyply/src/meta_molecule.py:325]: yeah we had some problems with sphinx that's why I have the habit of writing the full path. However, the problem has been resolved so this doc-string suggestion should be fine.

**Pull request merged**